### DISCUSSION TITLE
Ceremonias - Planning Meeting 16/06/2022 - Sprint 2

### DISCUSSION THREAD

**Discussion by Carolina-Bolatti** [ispc-programador2022/a2g3#25] [Mostrar y contar]
Buenos días!

He aquí cómo fue nuestra Reunión de Planning del 2do Sprint en el día 16/06/2022 por la mañana junto al PO @emilianoblancodev a través de la plataforma Meet!

La reunión se centró en ver lo nuevo a trabajar y a asignar. Para éste Sprint se decidió que trabajaremos sobre tres ejercitaciones más, y se les asignó a tres desarrolladores, que aún no fueron asignados anteriormente, éstas mismas. 

![WhatsApp Image 2022-06-16 at 10 11 09 AM](https://user-images.githubusercontent.com/105814132/174078263-e842a159-5dde-4d3a-ad0d-034cd717235a.jpeg)

**Comment by Raimun2000** [comment]
Buenísimo! Seguimos avanzando!!!  💪😀
